crates/stages/src/pipeline.rs

```diff
@@ -217,7 +206,6 @@ impl<DB: Database> Pipeline<DB> {
         // Sort stages by unwind priority
         let mut unwind_pipeline = {
             let mut stages: Vec<_> = self.stages.iter_mut().enumerate().collect();
-            stages.sort_by(|a, b| a.1.unwind_priority.cmp(&b.1.unwind_priority));
             stages.reverse();
             stages
         };
```

Comment:
unwind pipeline can be kept as an iterator `let unwind_pipeline = self.stages.iter_mut().rev();``